README.md

```diff
@@ -16,9 +16,9 @@ This workflow lets you manage your Pocket list with Alfred.
 - Background cache refresh
 - Supports notifications
 - Uses OAuth 2.0 to authorize the workflow
-- Saves your access_token securely in OS X's keychain
+- Saves your access_token securely in macOS keychain
 
 
 ## Credits
 
-This workflow uses [pocket](https://github.com/tapanpandita/pocket) and [alfred-workflow](https://github.com/deanishe/alfred-workflow).
+This workflow uses [pocket](https://github.com/tapanpandita/pocket) and [Alpynist](https://github.com/xilopaint/alpynist).
```

Comment:
The [link](https://github.com/xilopaint/alpynist) returns a 404 here. Is the repository private?

Reply:
The repository is not published yet, it's only local. The codebase, including the tests, is finished, but I still need to work on README and the documentation. Currently, you can find the latest version of Alpynist in my [Alfred PDF Tools](https://github.com/xilopaint/alfred-pdf-tools) workflow. For now you can credit me using the link of my profile.

Reply:
Was the repository published since then? The link is still dead.

Reply:
I no longer have plans to publish the repository as I switched from Alfred to Raycast. Alpynist is pretty solid and stable, though, and I'll continue to fix any reported library issues in the Alfred PDF Tools repository.